during_the_workshop.md

```diff
@@ -170,6 +170,20 @@ Scroll down this [documentation page](https://docs.docker.com/engine/reference/b
 1. After copying the file in, let's configure it as executable. On Linux, you do this by running a shell command e.g. `chmod +x ./my-file.sh`. Add a `RUN` command to the Dockerfile that makes the run.sh file executable.
 1. At the moment, nothing happens if you build and run a container based on this Dockerfile. You need to tell it what to do when starting the container. Add an `ENTRYPOINT` command that executes the `run.sh` file.
 
+<details markdown="1"><summary>Click here for answers</summary>
+
+1. `RUN apt-get install -y curl`
+    - This must be done before the `RUN curl ...` line.
+    - Or better yet, modify the existing line for jq instead: `RUN apt-get install -y jq curl`. This will installing both jq and curl. 
```

Comment:
```suggestion
    - Or better yet, modify the existing line for jq instead: `RUN apt-get install -y jq curl`. This will install both jq and curl. 
```
weird grammar